src/stores/DAOStores.js

```diff
@@ -22,8 +21,42 @@ import type {
   Tap,
 } from 'brewskey.js-api';
 
-import DAOApi from 'brewskey.js-api';
-import { Atom } from 'mobx';
+import DAOApi, { LoadObject } from 'brewskey.js-api';
+import { Atom, autorun } from 'mobx';
+
+export const waitForLoaded = <TValue>(
```

Comment:
How is this different from the one in the DAO?

Reply:
not much different, but it 1) works for stores methods( I think its more clean to use getMany() everywhere instead of Store.getMany() in some place and DAO.fetchMany() in another) 2) works for any store, so previosly in : `PourDAO.waitForLoaded( () => PourDAO.fetchMany())`  PourDAO.waitForLoaded worked only for PourDAO fetch methods... now we can use waitForLoaded, without perfix and it works just fine for any daostore.